tensorflow/tools/api/golden/v1/tensorflow.pbtxt

```diff
@@ -1340,6 +1340,10 @@ tf_module {
     name: "gather_nd"
     argspec: "args=[\'params\', \'indices\', \'name\', \'batch_dims\'], varargs=None, keywords=None, defaults=[\'None\', \'0\'], "
   }
+  member_method {
+    name: "generate_bounding_box_proposals"
```

Comment:
Internal review comments : 'The op needs a visibility:HIDDEN to avoid polluting the toplevel TF namespace'

Reply:
Please fix this @samikama

Reply:
Done